state/facade.go

```diff
@@ -45,6 +45,7 @@ type Facade interface {
 	AccountByNumber(number int32) *account.Account
 	ValidatorByAddress(addr crypto.Address) *validator.Validator
 	ValidatorByNumber(number int32) *validator.Validator
+	Validators() []*validator.Validator
```

Comment:
It's my fault that I couldn't explain it properly in #502. 
What we actually need is a list of all validator addresses:
```go
	Validators() []crypto.Address
```

Reply:
@b00f It's not `Validators()`, it's `ValidatorsAddress()`

Reply:
`ValidatorAddresses()` sounds better. Thanks